downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -593,6 +593,14 @@ public function main() returns error? {
 
 - Specified SSL as the preferred option when users provide options without SSL configuration. Additionally, introduced support for explicitly disabling SSL.
 
+#### `java.jms` package
+
+- Revamped Ballerina JMS protocol connector to the Swanlake standards.
```

Comment:
```suggestion
- Revamped the Ballerina JMS protocol connector to Swan Lake standards.
```

Since these are under `java.jms` package do we really need JMS protocol connector, Ballerina JMS connector, etc.?

Reply:
Fixed with [6aa565c093313e2eefd631432f8daf68f3b52896](https://github.com/ballerina-platform/ballerina-dev-website/pull/8972/commits/6aa565c093313e2eefd631432f8daf68f3b52896)